pkg/vpcmodel/detailedConn.go

```diff
@@ -98,13 +100,26 @@ func (e *detailedConn) subtract(other *detailedConn) *detailedConn {
 	return newDetailedConn(rspConn, otherConn, conn)
 }
 
+// returns the tcp responsive and non-tcp component
+func (e *detailedConn) tcpRspNonTCPComponent() *connection.Set {
+	return e.tcpRspEnable.Union(e.nonTCP)
+}
+
+// todo: remove once transformation is completed, rename enhanceString to string
```

Comment:
not sure I understand this `todo`, and why is `enhanceString` required? (I'd consider renaming it if required) .

Reply:
enhanceString is used to differentiate between printing the "main" - `tcp` responsive and `non-tcp` - component and the tcp non-responsive connection.
When the transformation is completed also for diff, string() should not be needed anymore - and `enhanceString()` can be renamed to `string()`

Reply:
ok, please rename and document the function

Reply:
done